Misc/NEWS.d/next/Library/2018-08-20-16-48-32.bpo-34441._zx9lU.rst

```diff
@@ -0,0 +1,2 @@
+Fix crash when an ``ABC``-derived class with non-callable ``__subclasses__`` is passed as the
+second argument to ``issubclass()``.
```

Comment:
\`\`issubclass()\`\` -> :func:\`issubclass\`

Reply:
Please add "Patch by Your Name." and wrap lines at 80 chars.

Reply:
Fixed, thank you.